pkg/secrets/plugins/file-system/plugin.go

```diff
@@ -0,0 +1,40 @@
+package filesystem
+
+import (
+	"os"
+
+	"get.porter.sh/porter/pkg/secrets"
+	"get.porter.sh/porter/pkg/secrets/plugins"
+	"github.com/hashicorp/go-hclog"
+)
+
+const PluginKey = secrets.PluginInterface + ".porter.file-system"
```

Comment:
Let's remove the hyphen (it's not really needed for this word anyway, its a single word. Just "filesystem" will work.